app/controllers/container_node_controller.rb

```diff
@@ -25,5 +25,13 @@ def launch_cockpit
     end
   end
 
+  def show_ad_hoc_metrics
+    if @record && @record.try(:ems_id)
+      ems = ExtManagementSystem.find(@record.ems_id)
```

Comment:
There has been a recent effort to make sure we always go through RBAC (see https://github.com/ManageIQ/manageiq-ui-classic/pull/779 for example)...

So I *think* this should be `ems = find_record_with_rbac(ExtManagementSystem, @record.ems_id)`.

@martinpovolny WDYT?